src/static/add_tta_match_result.html

```diff
@@ -3,35 +3,112 @@
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
 <head>
-  <title>Add a match result</title>
-  <script src="https://unpkg.com/vue@latest"></script>
-  <script src="https://unpkg.com/vue-select@latest"></script>
-  <link type="text/css" rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
-  <link type="text/css" rel="stylesheet" href="/static/styles.css">
-  <meta charset="utf-8">
-  <meta name="viewport" content="width=device-width">
-  <script src="/static/js/http.js" async=true></script>
-  <script src="/static/js/add_tta_match_result.js" async=true></script>
+    <title>Add a match result</title>
+    <script src="https://unpkg.com/vue@latest"></script>
+    <script src="https://unpkg.com/vue-select@latest"></script>
+    <link type="text/css" rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+    <link type="text/css" rel="stylesheet" href="/static/styles.css">
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <script src="/static/js/http.js" async=true></script>
+    <script src="/static/js/add_tta_match_result.js" async=true></script>
 </head>
 
 <body onload="onLoad()">
-  <h1>Add a TTA Match Result</h1>
-
-  <!--Container is hidden by display:none at the beginning, and will only be shown when initialized-->
-  <div id="container" class="container-fluid" style="width:80%; text-align:left; display:none;">
-    <h2>Select a Tournament:</h2>
-    <v-select id="tournament_selector" v-model="selected" :options="options"></v-select>
-    <h2>Select next player to add:</h2>
-    <v-select id="user_selector" v-model="selected" :options="options"></v-select>
-    <button class="btn-success" onclick="addUser()">Add Player</button>
-    <h2>Player Ranking</h2>
-    <div id="ranking">
-      <h3 v-for="player in ranking">
-        {{ player }}
-      </h3>
+    <h1>Add a TTA Match Result</h1>
+
+    <!--Container is hidden by display:none at the beginning, and will only be shown when initialized-->
+    <div id="container" class="container-fluid" style="width:80%; text-align:left; display:none;">
+        <h2>Select a Tournament:</h2>
+        <v-select id="tournament_selector" v-model="selected" :options="options"></v-select>
+        <h2>Select next player to add:</h2>
+        <v-select id="user_selector" v-model="selected" :options="options"></v-select>
+        <button class="btn-success" onclick="addUser()">Add Player</button>
+
+        <h2>Match Result:</h2>
+        <table>
+            <!--<tbody>-->
+            <tr>
+                <td></td>
+                <td>User</td>
+                <td>Score</td>
+            </tr>
+            <tr>
+                <td>Player 1</td>
+                <td>
+
+                    <div id="player_one">
```

Comment:
There are quite repeated and very hard-coded. Could you use Vue's v-for syntax to automatically generate these without deplicating html so much?